gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
-#Mon Feb 15 21:19:41 CET 2016
+#Fri Mar 18 15:48:56 CET 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.11-all.zip
-distributionSha256Sum=a1242e4db8f979998796b1844e608c2acf8f8f54df518bbb3d5954e52253ba71
\ No newline at end of file
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-bin.zip
```

Comment:
And to keep `-all` variant - it is a plugin - I heavily use Gradle source code during development.